mapit_mysociety_org/templates/api_keys/api_key_list.html

```diff
@@ -10,25 +10,27 @@
         <h2>{% trans "API Key" %}</h2>
         {% include 'account/_messages.html' %}
 
-        <p>To use your API key, include it in each API call as either an
+        <p>To use your API Key, include it in each API call as either an
```

Comment:
It looked like we use a capital "K" everywhere else on the page, like "Api Key" is a special thing. 😕